blocks/components/editable/index.js

```diff
@@ -296,12 +299,7 @@ export default class Editable extends wp.element.Component {
 		if ( focus ) {
 			element = [
 				<Fill name="Formatting.Toolbar" key="fill">
-					<Toolbar
-						controls={ formattingControls.map( ( control ) => ( {
-							...control,
-							onClick: () => this.toggleFormat( control.format ),
-							isActive: this.isFormatActive( control.format )
-						} ) ) } />
+					<FormatToolbar focusPosition={ this.state.focusPosition } formats={ this.state.formats } onChange={ this.changeFormats } />
```

Comment:
Yeah we put in under the link because we kept in mind that we might want to do something different above. There might be more reasons I don't remember though.